env.local

```diff
@@ -0,0 +1,6 @@
+REACT_APP_BACKEND_URL = "https://lessoner-project-2w3h.onrender.com"
```

Comment:
This file to be added into .gitignore. Do not include it in any PR

Reply:
Done